pom.xml

```diff
@@ -13,8 +13,8 @@
   <version>4.0.0-SNAPSHOT</version>
 
   <properties>
-    <stack.version>4.0.0-SNAPSHOT</stack.version>
-    <rabbitmq.client.version>5.4.2</rabbitmq.client.version>
+    <stack.version>4.0.0-milestone4</stack.version>
```

Comment:
why is this changed ?

Reply:
Stupid error on my part because I'm using the compiled jar in my projects - shouldn't have made it to the commit.